stable/pipelines/templates/buildplane-secret-k8s.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if and .Values.buildPlane.dynamic.provider.k8s.enabled ( not .Values.buildPlane.dynamic.provider.k8s.existingSecret ) }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "pipelines.fullname" . }}-dynamic-buildplane-creds-k8s
+  labels:
+    {{- include "pipelines.labels" . | nindent 4 }}
+type: Opaque
+data:
+  kubeconfig: {{ .Values.buildPlane.dynamic.provider.k8s.kubeconfig | quote }}
```

Comment:
Why is there no base64 encoding? I think it makes more sense to encode it to bade64 here instead of expecting the user to do it by themselves